docs/reference/remap/parse_key_value.cue

```diff
@@ -0,0 +1,52 @@
+package metadata
+
+remap: functions: parse_key_value: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The string to parse."
+			required:    true
+			type: ["string"]
+		},
+		{
+			name:        "key_value_delimiter"
+			description: "The string that separates the key from the value."
+			required:    false
+			default:     "="
+			type: ["string"]
+		},
+		{
+			name:        "field_delimiter"
+			description: "The string that separates each key/value pair."
+			required:    false
+			default:     " "
+			type: ["string"]
+		},
+	]
+	return: ["map"]
+	category: "Parse"
+	description: """
+		Parses a string in key value format.
+		Keys and values can be wrapped with a `"`. `"` characters within a delimited field can be escaped by `\\`.
+		"""
+	examples: [
+		{
+			title: "Successful match"
+			input: {
+				message: #"""
+					level=info msg="Stopping all fetchers" tag=stopping_fetchers id=ConsumerFetcherManager-1382721708341 module=kafka.consumer.ConsumerFetcherManager
+					"""#
+			}
+			source: #"""
+					. = parse_key_value(.message, field_delimiter=" ", key_value_delimiter="=")
```

Comment:
Might be good to remove the options in this example, since they are the default, and it helps make clear that this function is easy to use.

It's probably best to then add a second, more complicated example, that shows how to use the two extra options.